src/store/account/accountSlice.ts

```diff
@@ -271,7 +271,7 @@ const accountSlice = createSlice({
       },
     )
     builder.addCase(
-      updateSetting.pending,
+      updateSetting.fulfilled,
```

Comment:
@matthewcarlreetz I'm not sure we needed this to be pending? But if it is, then when the API call fails the app data still changes.

Reply:
I did it that way so that the ui could update without having to wait for the api response. Maybe we just handle failed instead of changing this to fulfilled?

Reply:
If we added and then removed it might feel weird to see the UI change and then change back when it fails? For me even with it at fulfilled it loaded quickly. I guess if the api is slow it might lag a second. What do you think, I'm leaning towards leaving it as fulfilled?

Reply:
My feeling is the immediate feedback is needed in the case that the api is slow. In the rare case that the api is down, I think it's ok to change the ui back if the call fails.